scripts/addb-py/chronometry/task_queue/task_queue.py

```diff
@@ -62,7 +62,8 @@ def task_add():
         return
 
     opt  = { 'enqueue_time': str(datetime.now()) }
-    task = io_workload_task((config, opt), priority=config['common']['priority'])
+    priority = config['common']['priority']
```

Comment:
if python interprets priority as a keyword, would it be better to use a different name, something like say task_priority?